vscode/src/circuit.ts

```diff
@@ -46,6 +60,7 @@ export async function showCircuitCommand(
   const compilerTimeout = setTimeout(() => {
     timeout = true;
     sendTelemetryEvent(EventType.CircuitEnd, {
+      simulated: false.toString(),
```

Comment:
Would it make sense to add a comment about how/why simulated starts false and may change in the circuit start/end calls for telemetry?